docs/gamejam/lessons/ocean.md

```diff
@@ -0,0 +1,52 @@
+
+# Game Jam in a Box
+
+![MakeCode Arcade Ocean Jam](/static/gamejam/jams/ocean/assets/preview.png)
+
+A game jam is a hackathon for building games! The "official" [MakeCode Arcade Ocean Jam](https://arcade.makecode.com/gamejam/ocean) runs from Nov 30, 2020 to Dec 11, 2020, but the game jam is really just a concentrated period of time for students to make games around a theme, so it can be run in any classroom, for any length of time--from an hour, to several weeks!
+
+## Rules
+
+The rules are lightweight and can be adjusted as necessary:
+1.  Your game must relate to the jam theme. Someone playing your game should be able to see how it relates to the theme without extra explanation.
+2.  Be creative!
+
+## Examples
+
+The theme for this game jam is **Ocean**. This could be games about swimming, or beaches, or marine life. Some real life examples of "ocean-themed" games might be Ecco the Dolphin, Sea of Thieves, or BioShock. Students should feel free to interpret the theme creatively! Some other game concepts that fit the theme include:
```

Comment:
Nit: bioshock is one of my favorites, but it's possibly a bit aggressive / if this is aimed at classrooms and school clubs it might not be something teachers would feel comfortable suggesting to students - maybe Subnautica as an alternate with somewhat similar thriller / survival feelings?

Reply:
updated!